### ISSUE TITLE
anything happen to com.google.android.gms:google-play-services lately

### ISSUE THREAD

**Issue by spyhunter99** [simpligility/maven-android-sdk-deployer#295]
I went to build today and found that the following artifact can no longer be resolved...

````
<groupId>com.google.android.gms</groupId>
<artifactId>google-play-services</artifactId>
<type>apklib</type>
<version>25.0.0</version>
```

after updating with the SDK manager and redeploying with this tool, i find that the artifact doesn't appear to exist anymore. Not sure what the replacement is or if my rig is broken

**Comment by mosabua** [comment]
Yeah... I wiped by SDK and reinstalled and lots of stuff is broken ... another fun update. Pull requests to fix things up are welcome..

**Comment by spyhunter99** [comment]
bummer

**Comment by mosabua** [comment]
Well.. I got the CI build green at least so it should be usable again for the core use cases like android.jar of various versions plus the repositories. I will work on add ons and extras soon..

**Comment by mosabua** [comment]
Play services is completely different now. Not sure where to get the apk from now to be honest. You would have to have an internal repo with the old artifact from a prior version deployed. If you need that I can see if I can find it on an old server.

**Comment by spyhunter99** [comment]
i love the fact that they just delete the old stuff. isn't the point of version'd artifacts is to be able to roll back?

**Comment by malachid** [comment]
They don't use repos correctly. They rename, delete, etc. If you tag a
build that depends on google repos, a year later, it is unlikely that it
can be repeated.

**Comment by mosabua** [comment]
Contact me via email .. I might have some of the needed artifacts..

**Comment by spyhunter99** [comment]
@mosabua any solutions? we've been broke for a while...and I can't even count how many man hours have been spent trying to get gradle to actually perform a release to central.

**Comment by spyhunter99** [comment]
looks like they removed the pom.xml from the jar...at least i think that's the case. the source is still there

**Comment by spyhunter99** [comment]
can we just make our own pom and carry on?

**Comment by Mohsenrezayii** [comment]
> I went to build today and found that the following artifact can no longer be resolved...
> 
> ```
> <groupId>com.google.android.gms</groupId>
> <artifactId>google-play-services</artifactId>
> <type>apklib</type>
> <version>25.0.0</version>
> ```
> 
> after updating with the SDK manager and redeploying with this tool, i find that the artifact doesn't appear to exist anymore. Not sure what the replacement is or if my rig is broken
